### PR TITLE
add fewest_tackles method along with helper methods_NT

### PR DESCRIPTION
**Pull request by NickTassinari**
This pr adds fewest tackles method along with helper methods that we can use for the most tackles methods potentially

### CODE REVIEW

File: lib/season_stats.rb
Comment (sarahgarlock) [inline lib/season_stats.rb:1]: Methods look good but still need tests in the spec

**Review by sarahgarlock — CHANGES_REQUESTED** [review]
Methods look good but still need tests in the spec

File: lib/season_stats.rb
Comment (NickTassinari) [reply lib/season_stats.rb:1]: oops

File: spec/stat_tracker_spec.rb
Comment (sarahgarlock) [inline spec/stat_tracker_spec.rb:1]: Thanks for adding that

**Pull request merged**